tips/api/tip_generator.py

```diff
@@ -140,18 +133,17 @@ def enrich_tip(tip):
             break  # only one enrichment per tip allowed
 
 
-def tips_generator(user_data, tips=None):
+def tips_generator(request_data={}, tips=None):
```

Comment:
Default arg wat mutable is (dus bijv {} of []) word at parse time geinitialiseerd, dus je moet deze constructie gebruiken:
```suggestion
def tips_generator(request_data=None, tips=None):
if request_data is None:
    request_data = {}
```

Reply:
Ah dus die wordt pas gebruikt wanneer er géén argument is? `tips_generator() != tips_generator(None)` ok.